### PR TITLE
Adding card parent component 

### PR DESCRIPTION
**Pull request by thusithak**
Fixes [#90](https://github.com/wso2/ux-theme-wso2/issues/90)

### CODE REVIEW

File: module/index.js [hunk 1]
```diff
@@ -7,3 +7,4 @@ export { Badge } from './components/Badge';
 export { Button } from './components/Button';
 export { Chip } from './components/Chip';
 export { Avatar } from './components/Avatar';
+export { Card } from './components/Card';
```
Comment (dakshika) [inline module/index.js]: Better to order ascending

File: module/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Card
+
+$card-spacer-y:            .75rem !default;
```
Comment (dakshika) [inline module/stylesheets/components/_card.scss]: why you are not using any of these ?

File: styleguide.config.js [hunk 3]
```diff
@@ -44,7 +44,8 @@ module.exports = {
       components: () => ([
         path.resolve(__dirname, 'module/components/Button', 'index.js'),
         path.resolve(__dirname, 'module/components/Chip', 'index.js'),
-        path.resolve(__dirname, 'module/components/Avatar', 'index.js')
+        path.resolve(__dirname, 'module/components/Avatar', 'index.js'),
+        path.resolve(__dirname, 'module/components/Card', 'index.js'),
```
Comment (dakshika) [inline styleguide.config.js]: place accending order

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/stylesheets/components/_card.scss (hunk 2 above)
Comment (thusithak) [reply module/stylesheets/components/_card.scss]: Removed unwanted sass variables

File: module/components/Card/Readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Card:
+
+```js
+<Card raised="true"></Card>
```
Comment (dakshika) [inline module/components/Card/Readme.md]: Isn't this <Card raised>

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/components/Card/index.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ * @returns Card
+ */
+class Card extends React.Component {
+    render() {
+        const {
+            className,
+            raised,
+            ...attributes
+        } = this.props;
+        const defaultClasses = (raised === 'true' ? 'card-raised' : null);
```
Comment (dakshika) [inline module/components/Card/index.js]: I think raised check is enough

File: module/components/Card/index.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * @ignore
+     */
+    children: PropTypes.node,
+};
+
+const defaultProps = {
+    className: 'card',
+    raised: false,
+    children: null,
```
Comment (dakshika) [inline module/components/Card/index.js]: this cannot  be null

**Review by dakshika — CHANGES_REQUESTED** [review]

**Pull request closed without merging**